.github/workflows/ci.yaml

```diff
@@ -133,4 +161,7 @@ jobs:
 
 
 env:
-   GRADLE_OPTS: -Dorg.gradle.configureondemand=false -Dorg.gradle.parallel=false -Dkotlin.incremental=false -Dorg.gradle.jvmargs="-Xmx3g -XX:MaxMetaspaceSize=756m -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8"
+   #   GRADLE_OPTS: -Dorg.gradle.configureondemand=false -Dorg.gradle.parallel=false -Dkotlin.incremental=false -Dorg.gradle.jvmargs="-Xmx3g -XX:MaxMetaspaceSize=756m -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8"
```

Comment:
This is mostly the same config as is already defined in the project's `gradle.properties`, except it disables some caching/optimisations. I think that re-enabling the config would help performance, and won't cause issues. And then since the rest of the config is already defined in the project, the whole thing can be removed.